Spigot-API-Patches/0190-Implement-tick-times-API.patch

```diff
@@ -0,0 +1,64 @@
+From 5589e305be48fb4a5cd6dbe650acba709f24a6cd Mon Sep 17 00:00:00 2001
+From: William Blake Galbreath <Blake.Galbreath@GMail.com>
+Date: Sun, 2 Feb 2020 03:28:56 -0600
+Subject: [PATCH] Implement tick times API
+
+
+diff --git a/src/main/java/org/bukkit/Bukkit.java b/src/main/java/org/bukkit/Bukkit.java
+index 388c2bbf..687be230 100644
+--- a/src/main/java/org/bukkit/Bukkit.java
++++ b/src/main/java/org/bukkit/Bukkit.java
+@@ -1603,6 +1603,25 @@ public final class Bukkit {
+     public static int getCurrentTick() {
+         return server.getCurrentTick();
+     }
++
++    /**
++     * Get the last 100 tick times (in nanos)
```

Comment:
maybe we should leave how many last ticks are actually in the array to its length in case it's ever changed for any reason

Reply:
If it ever changes the obfhelper will conflict and this can be updated. I feel its necessary to mention the length so the duration (5 seconds) is noted.

Reply:
yeah but ugh updating docs and breaking things potentially
technically if the average tickrate is above 50 it would be more than 5 seconds too